apps/twoopstracker/next.config.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
```

Comment:
If `twoopstracker` had Sentry integrated, lets enable it or add an issue to track it. I see `trolltracker-ui` project in the [dashboard](https://code-for-africa.sentry.io/projects/trolltracker-ui/?project=6096970).

Reply:
Tracked in #1113